### PR TITLE
Update license for consistency

### PR DESCRIPTION
**Pull request by dhmlau**
### Description

- Update the license to make "IBM copyright" instead of "StrongLoop copyright" for being consistent with the other modules
- update package.json to the right value for `slt copyright` command to run properly
- run `slt copyright` to update the copyright header of source code

Related to: strongloop-internal/scrum-apex#382

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -9,13 +13,13 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
...
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
```
Comment (virkt25) [inline LICENSE]: Need a new line here

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,10 @@
-MIT License
+Copyright (c) IBM Corp. 2012,2017. All Rights Reserved.
```
Comment (shimks) [inline LICENSE]: should this be 2012,2018? Same thing for the swagger connector

Comment (dhmlau) [reply LICENSE]: should be. thanks.

**Comment by crshnburn** [comment]
LGTM. Landed and thanks for the contribution @dhmlau

**Pull request merged**